pyproj/crs.py

```diff
@@ -19,11 +19,22 @@
 NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 """
-__all__ = ["CRS", "is_wkt"]
+__all__ = [
+    "CRS",
+    "CoordinateOperation",
+    "Datum",
+    "Ellipsoid",
+    "PrimeMeridian",
```

Comment:
Is there a reason that CoordinateSystem is not in this list? 
(because there is no way to create it from the class itself?)

Reply:
But I see below that you are including them in the sphinx docs (in the `_crs` namespace), so maybe we could add it to this namespace nevertheless? (so the namespace in sphinx is the same as for the other objects)

Reply:
> because there is no way to create it from the class itself?

Yes, that is correct. No user facing way to create it. So, only access is through the CRS class. I prefer not to include it for clarity of usage. 

> I see below that you are including them in the sphinx docs

I am including it so the user knows what they are getting from the CRS class. Just a reference for them when looking at the return types from the CRS class.